resources/k8s-artifacts/choreo-connect/rate-limiter-service.yaml

```
@@ -0,0 +1,41 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2023, WSO2 LLC. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+# service for rate-limiter
+
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: choreo-connect-rate-limiter
+  name: rate-limiter
+spec:
+  type: ClusterIP
```

```suggestion
  type: ClusterIP
  clusterIP: None
```
Let's use a headless service

---

fixed with: https://github.com/wso2/product-microgateway/pull/3281/commits/a20b1f2b377af726154bfa5eba80694b2ac7bfa6